tests/unit/lms/services/mailchimp_test.py

```diff
@@ -12,6 +13,14 @@
     factory,
 )
 
+#: A factory for course dicts as expected by the instructor_email_digest template.
+CourseDict = make_factory(
+    dict,
+    title=Sequence(lambda n: "Course {n}"),
+    num_annotations=3,
+    annotators=[sentinel.student_1, sentinel.student_2],
```

Comment:
Could randomise the number of annotations and annotators but it's not necessary (and it might even get complicated when you realise that `num_annotations` must be `>=` `len(annotators)` otherwise it's nonsensical).